TESTING.md

```diff
@@ -0,0 +1,41 @@
+# How to test
+
+## Install the new Promtail version
+
+* Choose a test cluster: `opsctl login gauss`
+* Suspend these 2 flux kustomizations:`flux suspend kustomization -n flux-giantswarm collection; flux suspend kustomization -n flux-giantswarm flux`
+
+### The "easy" way
+
+* Deploy the new Promtail version using opsctl (replace `test-branch` with your branch's name): `opsctl deploy --use-kubeconfig promtail-app@test-branch`
+
+### The "hard" way
+
+If for some reason you can't use the `opsctl deploy` command or don't want to:
+* Either create a draft PR or directly use your branch
+* Look at the build step `ci/circleci: package and push promtail chart`, step `Push chart archive to OCI registry app catalog`
+    * Take note of pushed image name
+* Edit the `promtail` app: `kubectl edit -n giantswarm promtail`
```

Comment:
`kubectl edit -n giantswarm promtail` => `kubectl edit app -n giantswarm promtail`

Reply:
I found myself many times in a situation where I had no choice but using the "hard way" to install the new version. That's why I think it's a good think to have it written for anyone who might encounter the same issue

Reply:
I'm fine with the curretn structure but I'd like if we have the same base template ;)

Reply:
That would be nice yes :)

Reply:
Created a PR here : https://github.com/giantswarm/giantswarm/pull/25200